db-server/index.js

```diff
@@ -125,6 +126,7 @@ function createServer(db) {
 
   api.get('/keyFetchToken/:id/verified', withIdAndBody(db.keyFetchTokenWithVerificationStatus))
   api.post('/tokens/:id/verify', withIdAndBody(db.verifyTokens))
+  api.post('/tokens/:id/verifyWith', withIdAndBody(db.verifyTokensWithMethod))
```

Comment:
Updated this endpoint name to `/tokens/:id/verifyWith` and db to `verifyTokensWithMethod` over `verifySessionWithMethod`. This made sense because it verifies more than session tokens. Open to other names as well.